linker/jvm/src/main/scala/org/scalajs/linker/backend/closure/ClosureLinkerBackend.scala

```diff
@@ -133,7 +148,12 @@ final class ClosureLinkerBackend(config: LinkerBackendImpl.Config)
 
   private def closureCompiler(logger: Logger) = {
     val compiler = new ClosureCompiler
-    compiler.setErrorManager(new LoggerErrorManager(logger))
+    /* FIXME We used to leverage `compiler.setErrorManager` to send errors and
+     * warnings to our `logger`. However, that API has been locked down a bit
+     * too much in the current version of GCC, so we cannot do that anymore.
+     * As it stands, errors and warnings will be send to the standard error.
+     */
+    //compiler.setErrorManager(new LoggerErrorManager(logger))
```

Comment:
I doubt this is going to get fixed especially if this shouldn't happen if there are no bugs. So I think just letting the errors go to stderr and removing this comment altogether is fine.

Reply:
Fixed.